File: 3220-count-tested-devices-after-test-operations/count-tested-devices-after-test-operations.py
class Solution:
    def countTestedDevices(self, batteryPercentages: List[int]) -> int:
        ans = 0
        for i in range(len(batteryPercentages)-1):
            if batteryPercentages[i] > 0:
                ans+=1
                for j in range(i+1,len(batteryPercentages)):
                    if batteryPercentages[j] > 0:
                        batteryPercentages[j]-=1
        if batteryPercentages[-1] > 0:
            ans+=1
        return ans

File: 3220-count-tested-devices-after-test-operations/count-tested-devices-after-test-operations.js
/**
 * @param {number[]} batteryPercentages
 * @return {number}
 */
var countTestedDevices = function(batteryPercentages) {
    let ans = 0
    for (let i=0;i<batteryPercentages.length;i++){
        if (batteryPercentages[i] > 0){
            ans+=1
            for(let j=i+1;j<batteryPercentages.length;j++){
                if(batteryPercentages[j]>0){
                    batteryPercentages[j]-=1
                }
            }
        }
    }
    if(batteryPercentages[batteryPercentages.legth-1] > 0){
        ans+=1
    }
    return ans
};